applications/nrf_desktop/README.rst

```diff
@@ -27,6 +27,42 @@ The application architecture is modular and event-driven.
 This means that parts of application functionality were separated into isolated modules that communicate with each other using application events.
 Application events are handled by the :ref:`event_manager`.
 Modules register themselves as listeners of events they need to react on.
+The application event can be submitted by multiple modules and it can have multiple listeners.
+
+The event-based communication for each application module is presented as a table that is included on the :ref:`module's documentation page <nrf_desktop_app_internal_modules>`.
+
++---------------------------------------+-------------+------------------------------------+--------------+-----------------------------------------+
+| Source Module                         | Input Event | This Module                        | Output Event | Sink Module                             |
++=======================================+=============+====================================+==============+=========================================+
+| The module that submits ``eventA``.   | ``eventA``  | The module described by the table. |              |                                         |
++---------------------------------------+             |                                    |              |                                         |
+| Other module that submits ``eventA``. |             |                                    |              |                                         |
++---------------------------------------+-------------+                                    +--------------+-----------------------------------------+
+| The module that submits ``eventB``.   | ``eventB``  |                                    |              |                                         |
++---------------------------------------+-------------+                                    +--------------+-----------------------------------------+
+|                                       |             |                                    | ``eventC``   | The module that reacts on ``eventC``.   |
++---------------------------------------+-------------+------------------------------------+--------------+-----------------------------------------+
+
+The table contains the following columns:
+
+Source Module
+   The module that submits a given application event.
+
+Input Event
+   An application event that is received by the module described by the table.
+
+This Module
+   The module described by the table.
```

Comment:
Maybe add something like:
`This is the module that is the target of the Input Events and the source of Output Events directed to the Sink Module.`

Reply:
done